alibi_detect/od/pytorch/mahalanobis.py

```diff
@@ -0,0 +1,113 @@
+from typing import Optional, Union
+
+import torch
+
+from alibi_detect.od.pytorch.base import TorchOutlierDetector
+
+
+class MahalanobisTorch(TorchOutlierDetector):
+    ensemble = None
```

Comment:
Is there a reason we've gone with `None` here rather than `False`?

Reply:
Nope, I've changed